packages/dso-toolkit/src/components/button-row/button-row.scss

```diff
@@ -20,4 +21,14 @@
       margin-left: 0;
     }
   }
+
+  &.emphasized {
```

Comment:
`.dso-emphasized`, alle classnames prefixen we met `dso`, behalve in Web Components met `shadow: true` of `scoped: true`.